### ISSUE TITLE
[TF.Keras]: plotting utilities

### ISSUE THREAD

**Issue by innat** [tensorflow/tensorflow#48641]
The plot function is defined as follows 

```
tf.keras.utils.plot_model(
    model,
    to_file="model.png",
    show_shapes=False,
    show_dtype=False,
    show_layer_names=True,
    rankdir="TB",
    expand_nested=False,
    dpi=96,
)
```

Following features would be nice (IMO) to have with it:
- Plot subgroup (e.g. form layer index 5 to 20 or by layer's name) of `model`. For example: if we plot `EfficientNet`, this plot utils will plot the whole model from first to last. But let's say for some reason we manipulate intermediate block from `layer_id: 10` to `layer_id:30` and want to check or visualize.

**Comment by nikitamaia** [comment]
Adding the contributions welcome label to this issue for further investigation by the community. If you are interested in working on this feature, please leave a comment and I will assign it to you. Thanks!

**Comment by Anupam-USP** [comment]
Hey I want to take up this issue, but I am new in this. Can you please elaborate the issue?

**Comment by ashutosh1919** [comment]
@nikitamaia , I have started working on this issue as part of PR #49860 . Can you please assign this issue to me?

**Comment by ashutosh1919** [comment]
@nikitamaia , I think we can close this issue since the change is merged in keras repo after split in [this](https://github.com/keras-team/keras/commit/d9e7b83c70e2117f40465b79e2e8201cea46d36a) commit as part of PR #49860 .

cc @qlzh727

**Comment by qlzh727** [comment]
Yes, this issue can be closed now, since https://github.com/keras-team/keras/commit/d9e7b83c70e2117f40465b79e2e8201cea46d36a has reach nightly build.
